test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/IntegrationTests.cs

```diff
@@ -58,7 +58,7 @@ public void Dispose()
         [SkipUnlessEnvVarFoundTheory(CollectorHostnameEnvVarName)]
         public void TraceExportResultIsSuccess(OtlpExportProtocol protocol, string endpoint, ExportProcessorType exportProcessorType, bool forceFlush)
         {
-#if NETCOREAPP3_1
+#if NET6_0_OR_GREATER
```

Comment:
Does the old NETCOREAPP3_1 apply to .NET 6 or greater?

Reply:
it does not - this was a mistake.